main/views.py

```diff
@@ -1,5 +1,13 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.contrib.auth import logout
 
 
 def homePage(request):
+    if request.user.is_authenticated:
+        return render(request, 'main/loggedInPage.html')
     return render(request, 'main/homePage.html')
+
+
+def logout_view(request):
```

Comment:
Do we need this? we use `LogoutView` [here](https://github.com/redhat-beyond/HighTeach/blob/c217ffd45a7f20c43e9469da20943129ec1b04cc/HighTeach/urls.py#L24)

Reply:
This view is no longer needed, fixed